src/BinaryParsers/ElfBinary/Dwarf/DwarfCompilationUnit.cs

```diff
@@ -250,6 +257,7 @@ private void ReadData(IDwarfBinary dwarfBinary, DwarfMemoryReader debugData, Dwa
                             break;
 
                         case DwarfFormat.ImplicitConst:
+                            attributeValue.Value = descriptionAttribute.Value;
```

Comment:
>Value [](http://example.com/codeflow?start=72&length=5)

Now that we properly obtain the implicit const value from the debug abbreviations table, we can populate it here.